### DISCUSSION TITLE
failed in fleet agent final_pipeline field [created] not present as part of path [event.created]

### DISCUSSION THREAD

**Discussion by pietjuu** [Security-Onion-Solutions/securityonion#12418] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

96

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I have installed securityonion 2.4.40 and I dont get any syslog logs. 
My test setup is build with a server that runs esxi 8. On that server there are 3 virtual Machines: 1 SecurityOnion, 2 windows desktop, 3 debian webserver.

I noticed that in my dashboard there is a lot of syslogs, elastic_Agent.filebeat and other elastic logs from the securityunion are missing. I did configure the syslog firewall to allow all traffic on my network (allow 192.168.159.0/27)
How can i get the syslog data in my securityonion dashboards?

I tried to install the agent again on the Union but i got the message that it is already installed so I assume that it is working.
Below are some screenshots that hopefully help!

![afbeelding](https://github.com/Security-Onion-Solutions/securityonion/assets/60645715/238ad9ed-9305-44e8-ba68-db29c449da96)

![afbeelding](https://github.com/Security-Onion-Solutions/securityonion/assets/60645715/c8567d00-c753-4070-b58e-71c1f5160736)

![afbeelding](https://github.com/Security-Onion-Solutions/securityonion/assets/60645715/3964c9c6-a2e1-4ae9-bca5-c7d197eed76e)

![afbeelding](https://github.com/Security-Onion-Solutions/securityonion/assets/60645715/77d6ceb9-cfd8-4cd7-8cdf-c1b07a273d62)

FYI: salt status returns that it is running as PID and was started at [Timestamp] but it wasnt able to compile the data. 

![afbeelding](https://github.com/Security-Onion-Solutions/securityonion/assets/60645715/21336460-fa71-4280-9610-e70c1bef7b99)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by pietjuu** [comment]
After rerunning the command so-checkin (as root user and not like sudo so-checkin) it returned fully healthy 

![afbeelding](https://github.com/Security-Onion-Solutions/securityonion/assets/60645715/cd9858cf-c652-429f-82bb-4465d44c83e8)

**Comment by pietjuu** [comment]
What maybe the problem could be is this?
I have no clue but i found this in /var/log/messages
![afbeelding](https://github.com/Security-Onion-Solutions/securityonion/assets/60645715/07659150-136a-4019-ac14-aaf7608752f7)

**Comment by ryan-donat** [comment]
I am also having this issue. Similar specs (SO iso and status, but this is a distributed setup), all docker containers appear healthy. I believe this error is also preventing my elastic agents from working correctly. On fleet, all dashboards and elastic point to healthy status. The same error:   "failed in Fleet agent final_pipeline: field [created] not present as part of path [event.created]" is observed on all of my sensor nodes.

Please let me know if you figure anything out.

**Reply by pietjuu** [reply]
Do you have also the same error in the log files like I have?

**Reply by ryan-donat** [reply]
No I do not see that error anywhere in /var/log/messages

**Reply by rigbyst** [reply]
I am running into the same issue, however I have syslogs pointing to an elastic agent which sends it to SO Manager.

**Reply by ryan-donat** [reply]
@rigbyst How did you go about doing that?

**Reply by rozahp** [reply]
I have the exact same issue after ISO install on 2 systems: Eval and Standalone. 

Tried a lot of trick without any result.

I did an upgrade to the latest version 2.4.60 but that didn't help. 

Could it have to do with DNS? Just thinking loud...

**Comment by alveare1** [comment]
I am seeing the same issue. I wish I had more to add. I am also on 2.4.60 distributed setup.

**Comment by foxman0719** [comment]
Even if I had copied it, I have the same problem as I described it to our friend, can anyone from the community help us? 
I am also on the distributed configuration 2.4.60 which was supposed to solve the problem and nothing.
